src/Traits/ParsesCountries.php

```diff
@@ -15,15 +14,9 @@ trait ParsesCountries
      */
     public static function isValidCountryCode($country)
     {
-    	$iso3166 = new ISO3166;
+        $supportedRegions = app('libphonenumber')->getSupportedRegions();
```

Comment:
I prefer to omit helper functions in packages. Please use `PhoneNumberUtil::getInstance()` instead.